### PR TITLE
feat(sqlalchemy.py): add support for sqlalchemy

### PR DESCRIPTION
**Pull request by ranrib**
Screen Shot 2020-08-27 at 16 02 12

### CODE REVIEW

File: epsagon/utils.py [hunk 1]
```diff
@@ -29,6 +29,20 @@
...
 
+def patch_once(patch_module, patch_name, wrapper):
+    """
+    Patches a module once by setting `EPSAGON_MARKER` on the wrapped object
+    :param patch_module: module to patch
+    :param patch_name: attribute name to path
+    :param wrapper: new wrapper to set
+    :return: None
+    """
+    (_, _, original) = wrapt.resolve_path(patch_module, patch_name)
```
Comment (maorlx) [inline epsagon/utils.py:40]: no need for the ( )

File: epsagon/events/sqlalchemy.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+        # override event type with the specific DB type
+        self.resource['type'] = database_connection_type(
+            instance.bind.url.host,
+            self.RESOURCE_TYPE
+        )
+
+        if exception is not None:
+            self.set_exception(exception, traceback.format_exc())
+
```
Comment (maorlx) [inline epsagon/events/sqlalchemy.py:53]: don't we wanna save the response? (if metadata only is false)

Comment (ranrib) [reply epsagon/events/sqlalchemy.py:53]: There's nothing interesting in the response here

File: epsagon/utils.py (hunk 1 above)
Comment (ranrib) [reply epsagon/utils.py]: Thanks, fixed!

File: epsagon/events/sqlalchemy.py (hunk 2 above)
Comment (maorlx) [reply epsagon/events/sqlalchemy.py:53]: right, we just catch init & close sessions

**Review by maorlx — APPROVED** [review]

**Pull request merged**